test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/DeferThrowsErrorTest.kt

```diff
@@ -0,0 +1,62 @@
+package graphql.nadel.tests.next.fixtures.defer
+
+import graphql.nadel.NadelExecutionHints
+import graphql.nadel.tests.next.NadelIntegrationTest
+
+open class DeferThrowsErrorTest : NadelIntegrationTest(
```

Comment:
What's the difference between this test and ` DeferReturnsErrorTest` ?

Reply:
I think I was meaning to return a DataFetcherResult with an error as that was the case that was failing before your change in graphql-java (https://github.com/graphql-java/graphql-java/pull/3640). 

I think I was supposed to do:
```
                           .dataFetcher("slow") { env ->
                                DataFetcherResult.newResult<String>()
                                    .error(
                                        GraphqlErrorBuilder.newError()
                                        .message("An error occurred while fetching 'slow'")
                                        .build()
                                    )
                                    .build()
                            }
```

I will delete this file and add in the new test later when we have updated graphql-java